darshan-util/pydarshan/tests/input/__init__.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+def installed_path(filename):
+    cwd = os.path.dirname(__file__)
+    return os.path.join(cwd, filename)
```

Comment:
we could perhaps abstract this to avoid duplication if we stick with this approach to making data files available